### ISSUE TITLE
About PatchGAN for CycleGAN

### ISSUE THREAD

**Issue by DISAPPEARED13** [junyanz/pytorch-CycleGAN-and-pix2pix#1368]
Hi, Professor Zhu.
I notice that in the 3-layer discriminator definiation as PatchGAN, but I found that while training, just feed the **whole pictures** in D(), but PatchGAN will make a **sampling metrix** or sth, why?
Thanks a lot.

Best, D.P.13

**Comment by junyanz** [comment]
Yeah, PatchGAN will produce an output map given a whole image. You do not need to produce patches explicitly.
